### PR TITLE
SW-1801 Use same default date for withdraw modal as other modals like create accession

### PR DESCRIPTION
**Pull request by karthikbtf**
See bug video here https://www.loom.com/share/538d106e11b84e9e9345a03a967ac19e

- update util to use same code that other modals use

### CODE REVIEW

File: src/utils/date.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { getDateDisplayValue } from '@terraware/web-components/utils';
-import moment from 'moment';
 export default getDateDisplayValue;
 
 export const getTodaysDateFormatted = () => {
-  return moment().format('YYYY-MM-DD');
```
Comment (karthikbtf) [inline src/utils/date.tsx:6]: not sure what moment was returning maybe not utc

Comment (karthikbtf) [reply src/utils/date.tsx:6]: other modals use default today's date as `getDateDisplayValue(Date.now())`
open to other suggestions but prefer one code fragment for consistency

Comment (karthikbtf) [reply src/utils/date.tsx:6]: @constanzauanini fyi

Comment (constanzauanini) [reply src/utils/date.tsx:6]: We should delete this helper and just use getDateDisplayValue(Date.now()) everywhere. But this is fine for now if we don't want to replace it in this PR

**Review by constanzauanini — APPROVED** [review]

File: src/utils/date.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/utils/date.tsx:6]: I was thinking we could replace all usages of `getDateDisplayValue(Date.now())` with `getTodaysDateFormatted()` -  to be more high level. Just want to make sure you are ok with this plan of not using the `moment` based code.

Comment (constanzauanini) [reply src/utils/date.tsx:6]: Yes, Date is js native, so I think less conversions is better

**Pull request merged**